src/App.tsx

```diff
@@ -1,21 +1,31 @@
 import React from "react";
+import { Routes, Route } from "react-router-dom";
```

Comment:
did you forget to `yarn add react-router-dom`?

Reply:
Did `yarn add react-router-do` and did another PR